schema/source.yml

```diff
@@ -36,6 +36,21 @@ properties:
       - 3
       - 4
       - 5
+  opengovdata:
+    title: "Open government data principles"
+    type: array
```

Comment:
Maybe here object should be `object`? And it could look like this:

```yaml
opengovdata:
  principle1: true
  principle2: false
```

Not sure, why you want `comment` here? Could you give an example why it would be needed?

Reply:
principles are abstract, and it would be great to explanation assessment qualitatively

Reply:
We could go with simpler model to begin with.
 ```opengovdata:
    title: "Open government data principles"
    type: object```
would that be enough?

Reply:
At leas you should specify which principles are possible. And if this is going to be only boolean values, then maybe array would be better option:

```yaml
opengovdata:
  - principle1
  - principle2
```

Should could look like this:

```yaml
opengovdata:
  title: "Open government data principles"
  type array
  items:
    enum:
      - principle1
      - principle2
```